subprojects/docs/src/docs/userguide/dep-man/03-controlling-transitive-dependencies/resolution_rules.adoc

```diff
@@ -37,10 +37,10 @@ include::sample[dir="snippets/dependencyManagement/customizingResolution-resolut
 include::sample[dir="snippets/dependencyManagement/customizingResolution-resolutionStrategy/kotlin",files="build.gradle.kts[tags=custom-versioning-scheme]"]
 ====
 
-[[sec:blacklisting_version]]
```

Comment:
Should we go the extra mile and see if we can keep incoming links to that ID end up resolving to the new one?

Reply:
No idea how to do this, though.